server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -1,27 +1,92 @@
-#!/usr/bin/env python
-import sys
-import os
-import shutil
-for line in sys.stdin:
-    tarball = line.split(": ")[1]
-    try:
-        src = "TO-SYNC/%s" % tarball
-        des = "SYNCED"
-        shutil.move(src, des)
-    except Exception as e:
-        print (e)
-        continue
-    try:
-        os.remove('%s' % tarball)
-    except Exception as e:
-        print (e)
-    try:
-        os.remove('%s.md5' % tarball)
-    except Exception as e:
-        print (e)
-    name = ('.prefix/prefix.%s' % tarball.strip('.tar.xz'))
-    if name:
-        try:
-            os.remove(name)
-        except Exception as e:
-            print (e)
+#! /bin/bash
+
+# load common things
+opts=$SHELLOPTS
+case $opts in
+    *xtrace*)
+        dir=$(dirname $(which $0))
+        PROG=$(basename $(which $0))
+        ;;
+    *)
+        dir=$(dirname $0)
+        PROG=$(basename $0)
+        ;;
+esac
+
+remotearchive="/pbench/archive/fs-version-001"
```

Comment:
Move it to after sourcing of pbench_base.sh and use $ARCHIVE instead of the hard-coded string.